examples/coappearances/coappearances.flatdata

```diff
@@ -129,5 +129,26 @@ archive Graph {
 
     // All strings contained in the data separated by '\0'.
     strings: raw_data;
+
+    // Optional archive containing calculated statistics.
+    @optional
+    calculated_data : archive CalculatedData;
```

Comment:
How about naming it "statistics" with type "Statistics"?

Reply:
Indeed, makes sense.